src/internal/ppsutil/util.go

```diff
@@ -114,43 +112,27 @@ func GetLimitsResourceList(limits *pps.ResourceSpec) (*v1.ResourceList, error) {
 	return getResourceListFromSpec(limits)
 }
 
-// GetPipelineInfoAllowIncomplete retrieves and returns a PipelineInfo from PFS,
-// or a sparsely-populated PipelineInfo if the spec data cannot be found in PPS
-// (e.g. due to corruption or a missing block). It does the PFS
-// read/unmarshalling of bytes as well as filling in missing fields
-func GetPipelineInfoAllowIncomplete(pachClient *client.APIClient, ptr *pps.StoredPipelineInfo) (*pps.PipelineInfo, error) {
-	result := &pps.PipelineInfo{}
+// GetPipelineDetails etrieves and returns a PipelineInfo from PFS,
```

Comment:
etrieves

Reply:
I think this comment needs to be updated if it now either returns the details or errors.